pkg/skaffold/survey/config.go

```diff
@@ -39,7 +44,29 @@ var (
 		URL: hatsURL,
 	}
 	// surveys contains all the skaffold survey information
-	surveys = []config{hats}
+	surveys = []config{
+		hats,
+		{
+			id:       helmID,
+			startsAt: time.Date(2021, time.July, 11, 0, 0, 0, 0, time.UTC),
+			expiresAt: time.Date(2021, time.August,
+				14, 00, 00, 00, 0, time.UTC),
+			promptText: "Skaffold is working on revamping helm deployer support. Help us understand your helm deployer use case with our 2-minute anonymous survey",
+			isRelevantFn: func(cfgs []util.VersionedConfig, _ sConfig.RunMode) bool {
```

Comment:
wdyt about replacing `isRelevantFn` with an explicit type `survey.category` where:

```go
type category interface {
    valid([]util.VersionedConfig, sConfig.RunMode) bool
}

type categories []category

func (s categories) valid(c []util.VersionedConfig, m sConfig.RunMode) bool{
    for _, survey := range s {
        if !survey.valid(c, m) {
            return false
        }
    }
    return true
}
```
so we can arbitrarily compose them for more target surveys

```go
surveys = []config{
    {
        id: multiConfigHelmUsers,
        category: categories([]category{usesHelm(), isMultiConfig()})
    }
}
```

Reply:
sounds like a good idea.  Shd we do this when we do have to re-use?